language/examples/oss-samples/langchain/question_answering_with_large_documents_langchain.ipynb

```diff
@@ -0,0 +1,1131 @@
+{
```

Comment:
<p>May want to consider mentioning Matching Engine as a way for users to go beyond in-memory vector dbs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>done</p>